apps/browser/components/Disclaimer.tsx

```diff
@@ -0,0 +1,52 @@
+import clsx from "clsx";
+import Button from "./Button";
+import Logo from "./Logo";
+
+interface DisclaimerProps {
+  handleDisclaimerSelect: (accept: boolean) => void;
+  className?: string;
+}
+
+const Disclaimer = ({ handleDisclaimerSelect, className }: DisclaimerProps) => {
+  return (
+    <div
+      className={clsx(
+        "fixed bottom-4 z-50 w-full px-2 md:bottom-28",
+        className
+      )}
+    >
+      <div
+        className={clsx(
+          "animate-slide-up flex w-full max-w-[40rem] max-w-[40rem] items-center items-center justify-between justify-between space-x-2 space-x-2 self-center self-center rounded-xl rounded-xl border-2 border-2 border-cyan-500/70 border-cyan-500/70 bg-gradient-to-b bg-gradient-to-b from-cyan-500/50 from-cyan-500/50 to-cyan-700/50 to-cyan-700/50 px-2 py-2.5 text-xs text-white opacity-0 shadow-xl backdrop-blur-[4px] md:px-4 md:text-sm",
+          className
+        )}
+      >
```

Comment:
Both the top level div, and the second top-most div share the same `className` prop; is it intentional?

Reply:
nice catch, fixing it now